pymongo/message.py

```diff
@@ -1127,7 +1127,7 @@ class _EncryptedBulkWriteContext(_BulkWriteContext):
 
     def __batch_command(
         self, cmd: MutableMapping[str, Any], docs: List[Mapping[str, Any]]
-    ) -> Tuple[Dict[str, Any], List[Mapping[str, Any]]]:
+    ) -> Tuple[MutableMapping[str, Any], List[Mapping[str, Any]]]:
```

Comment:
NIT: Not sure if you wanted to do this for all of the typing parts (mainly for consistency), but thoughts on converting all of the `typing.[built-ins] -> [built-ins]` in another task?

That also balloons scope here so I wouldn't want to bloat this PR. 
```suggestion
    ) -> tuple[MutableMapping[str, Any], list[Mapping[str, Any]]]:
```

Reply:
Ah, I had forgotten about `tuple`.  I agree this should be done as part of https://jira.mongodb.org/browse/PYTHON-3905, which I should have done first.

Reply:
Python 3.9 is the oldest version to support the use of the literal generics such as `dict` without the future annotations import. I recall that we decided against using the literals while we still supported older versions.

Reply:
Right, but we're going to use `from __future__ import annotations` in all files, and there is not harm in continuing to do so indefinitely.